diff --git a/examples/dynsub/CMakeLists.txt b/examples/dynsub/CMakeLists.txt
index db3f76a9..03840bd6 100644
--- a/examples/dynsub/CMakeLists.txt
+++ b/examples/dynsub/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(dynsub LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(dynsub LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   find_package(CycloneDDS REQUIRED)
diff --git a/examples/helloworld/CMakeLists.txt b/examples/helloworld/CMakeLists.txt
index ac0a3427..c7292d5d 100644
--- a/examples/helloworld/CMakeLists.txt
+++ b/examples/helloworld/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(helloword LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(helloword LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   # Find the CycloneDDS package.
diff --git a/examples/listtopics/CMakeLists.txt b/examples/listtopics/CMakeLists.txt
index 39034874..42e14a36 100644
--- a/examples/listtopics/CMakeLists.txt
+++ b/examples/listtopics/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(listtopics LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(listtopics LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   find_package(CycloneDDS REQUIRED)
diff --git a/examples/roundtrip/CMakeLists.txt b/examples/roundtrip/CMakeLists.txt
index a7d0a16e..916187ec 100644
--- a/examples/roundtrip/CMakeLists.txt
+++ b/examples/roundtrip/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(roundtrip LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(roundtrip LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   # Find the CycloneDDS package.
diff --git a/examples/shm_throughput/CMakeLists.txt b/examples/shm_throughput/CMakeLists.txt
index df30cb60..25799c99 100644
--- a/examples/shm_throughput/CMakeLists.txt
+++ b/examples/shm_throughput/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(shm_throughput LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(shm_throughput LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   # Find the CycloneDDS package.
diff --git a/examples/throughput/CMakeLists.txt b/examples/throughput/CMakeLists.txt
index 5d4e4307..17ad2757 100644
--- a/examples/throughput/CMakeLists.txt
+++ b/examples/throughput/CMakeLists.txt
@@ -9,8 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-project(throughput LANGUAGES C)
 cmake_minimum_required(VERSION 3.16)
+project(throughput LANGUAGES C)
 
 if(NOT TARGET CycloneDDS::ddsc)
   # Find the CycloneDDS package.
